keps/sig-node/4216-image-pull-per-runtime-class/README.md

```diff
@@ -0,0 +1,611 @@
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: RuntimeClassInCRICalls
+  - Components depending on the feature gate: kubelet
+- [x] Other
+  - Describe the mechanism: `N/A`
+  - Will enabling / disabling the feature require downtime of the control
+    plane? `No`
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? `No`
+
+###### Does enabling the feature change any default behavior?
+
+No. Aim of this KEP to ensure default behavior is not changed. Any changes to CRI and kubelet will be under feature gate as mentioned in earlier sections.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. All the code changes needed will be under feature gate `RuntimeClassInCriCalls` that will be off by default for alpha.
+Default exisiting behavior of kubelet will be ensured.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+It should be a no-op
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, this test will be added for alpha.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+`Note:` This section will be filled once we move to beta.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Image pulls for non-default runtime class goes through. Alternately, one could check the status of the feature gate.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Image pull goes through successfully when non-default runtimehandler is used.
+For example, if host is a windows node, user should be able to pull images for hyperV isolation even if the container image OSVersion does not match the host OS version.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Exisiting behavior of kubelet should not change when runtime class is not speified when image is pulled.
```

Comment:
s/Exisiting/Existing/

there are a bunch of spelling errors / typos throughout the KEP. You may want to run it through a markdown spell checker. There are a few out there, I have used this one and it's kinda slow but works well enough: https://hub.docker.com/r/tmaier/markdown-spellcheck/